app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivity.kt

```diff
@@ -3,16 +3,22 @@ package org.oppia.android.app.devoptions
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import org.oppia.android.R
 import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.drawer.KEY_NAVIGATION_PROFILE_ID
+import javax.inject.Inject
 
 /** Activity for Developer Options. */
 class DeveloperOptionsActivity : InjectableAppCompatActivity() {
+  @Inject lateinit var developerOptionsActivityPresenter: DeveloperOptionsActivityPresenter
   private var internalProfileId = -1
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
     internalProfileId = intent.getIntExtra(DEVELOPER_OPTIONS_ACTIVITY_PROFILE_ID_KEY, -1)
```

Comment:
We're setting the internalProfileId with `KEY_NAVIGATION_PROFILE_ID ` and then retrieving it using `DEVELOPER_OPTIONS_ACTIVITY_PROFILE_ID_KEY`. Shouldn't it be the same key in both cases?

Reply:
Yes. I realised that when I was working on PR #3331. I have corrected this there.

Reply:
Done.